<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbLoad">
        <property name="text">
         <string>&amp;Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDeselect">
        <property name="text">
         <string>Deselect all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPlot">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QTableWidget" name="twAssets">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item>
       <widget class="QChartView" name="graphicsView_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="DChartView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="rbDateRange6m">
            <property name="text">
             <string>6M</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbDateRange1m">
            <property name="text">
             <string>1M</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbDateRange3m">
            <property name="text">
             <string>3M</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QRadioButton" name="rbDateRange5y">
            <property name="text">
             <string>5Y</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbDateRange3y">
            <property name="text">
             <string>3Y</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbDateRange12m">
            <property name="text">
             <string>1Y</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbDateRangeAll">
            <property name="text">
             <string>All</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QRadioButton" name="rbYAbsolute">
            <property name="text">
             <string>Absolute</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbYPercentage">
            <property name="text">
             <string>Percentage</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QRadioButton" name="rbPlotAV">
            <property name="text">
             <string>AV</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_3</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbPlotAVxUnits">
            <property name="text">
             <string>AV*Units</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_3</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbPlotTotal">
            <property name="text">
             <string>Total</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_3</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbPlotUP">
        <property name="text">
         <string>unit price</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPlotUPP">
        <property name="text">
         <string>Unit price % (v.s.  disp. origin.)</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPlotV">
        <property name="text">
         <string>Value</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPlotVP">
        <property name="text">
         <string>Value % (v.s. book cost)</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPlotSV">
        <property name="text">
         <string>Sum value</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPlotSVP">
        <property name="text">
         <string>Sum value % (v.s. book cost)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DChartView</class>
   <extends>QGraphicsView</extends>
   <header>dcharts.h</header>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_4"/>
 </buttongroups>
</ui>
